apps/src/code-studio/components/libraries/LibraryIdCopier.jsx

```diff
@@ -21,7 +21,7 @@ export default class LibraryIdCopier extends React.Component {
           type="text"
           ref={channelId => (this.channelId = channelId)}
           onClick={event => event.target.select()}
-          readOnly="true"
+          readOnly={true}
```

Comment:
Have no big context about the whole PR, just a smal comment re this line.
` readOnly={true}` can also be simplefied to just `readOnly`